packages/solo/src/vat-http.js

```diff
@@ -123,8 +128,17 @@ export function buildRootObject(vatPowers) {
       }
 
       if (decentralObjects) {
+        // Assign the complete decentralObjects to `agoric`.
         Object.assign(replObjects.agoric, decentralObjects);
         doneLoading(['agoric']);
+
+        // Prune out the demo governance stuff from `home`; they're noisy.
```

Comment:
I made a documentation note in #3154 that this means `home` no longer subsumes `agoric`.